.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: input
+    id: operating_system
+    attributes:
+      label: OS
+      description: Operating system and version (eg Ubuntu 20.04, OSX Catalina)
+      placeholder: linux
+
+  - type: architecutre
+    id: cpu_arch
+    attributes:
+      label: CPU Architecture
+      description: CPU Architecture (eg: x86, x86-64, arm, etc).
+      placeholder: linux
+
+  - type: input
+    id: redis_version
+    attributes:
+      label: Redis Server Version
+      description: What version of redis-server are you running (redis-server --version)
+      placeholder: latest
+
+  - type: input
+    id: redis_module_version
+    attributes:
+      label: Redis Module Version
+      description: What version of this redis module was installed?
+      placeholder: latest
+
+  - type: input
+    id: was_redis_docker
+    attributes:
+      label: Docker Instllatio
+      description: What version of this redis module was installed?
+      placeholder: latest
+
+  - type: checkboxes
+    id: installmethod
+    attributes:
+      label: Docker installation
+      description: Check this box, if you installed the release via a docker.
+      options:
+        - label: I installed the dockerhub release
+
+  - type: textarea
+    id: whathappened
+    label: What happened?
```

Comment:
Do we want to have checkboxes for scary stuff, like crash, loss-of-data, hang/not-responding, so it could be easily found?

Reply:
And what about milestone?

Reply:
I think milestone is covered by version. We could definitely do the checkbox data loss thing, but given that someone is going to use words to describe _something_ I looked at it like duplication.